### DISCUSSION TITLE
Frage

### DISCUSSION THREAD

**Discussion by beatagerger** [frenck/home-assistant-config#61] [General]
Hallo Gemeinde!

Ich verstehe noch immer nicht: 
wozu ein "Home Assistant", wobei diese auf Raspi läuft. (hässliches Ding, auf Ubuntu Basis)

Kann mir das bitte jemand sagen?

**Comment by frenck** [comment]
? Ich benutze keinen Raspberry Pi...

**Comment by beatagerger** [comment]
I am madam ....
then how do you run MQTT? it's for Raspi ... (Ubuntu)

Gesendet von Mail<https://go.microsoft.com/fwlink/?LinkId=550986> für Windows 10

Von: Franck Nijhof<mailto:notifications@github.com>
Gesendet: Freitag, 12. Februar 2021 17:25
An: frenck/home-assistant-config<mailto:home-assistant-config@noreply.github.com>
Cc: beatagerger<mailto:beatagerger@msn.com>; Author<mailto:author@noreply.github.com>
Betreff: Re: [frenck/home-assistant-config] Frage (#61)


? Ich benutze keinen Raspberry Pi Sir...

**Reply by frenck** [reply]
> then how do you run MQTT?

I do not use MQTT.

>  it's for Raspi ... (Ubuntu)

As I already told you, I don't use a Raspberry Pi.
